CONTRIBUTING.rst

```diff
@@ -280,13 +280,15 @@ Imports
   reported by our flake8 plugin are not always optimal in illuminating the
   nature of the violations.
 
-* The one violation *not* addressable via PyCharm is our requirement that
-  single-symbol ``from`` imports be wrapped the same as multi-symbol ones.
-  Currently, this must be corrected manually. Vim users may find the following
-  macro convenient for this purpose:
+* There are two known cases wherein PyCharm cannot automatically correct import
+  violations. First, modules imported from within the `scripts` directory are
+  not subjected to re-ordering when out of order. Second, PyCharm refuses to
```

Comment:
This is a very optional suggestion.

```suggestion
  not ordered. Second, PyCharm refuses to
```

Reply:
I don't like the way this sounds with passive voice but with active I think it's a good basis for an improvement.